### ISSUE TITLE
numpy and nibabel are imported when running packages/build_packages.py

### ISSUE THREAD

**Issue by ltirrell** [freesurfer/freesurfer#553]
### Platform details:

- FreeSurfer version: current dev (commit 2194c58)
- Operating system name and version: ubuntu 16 docker container

### Summary
The `packages/build_packages.py` script requires `numpy` and `nibabel` due to an `import *` within the the `python/freesurfer/__init__.py` file.

### Actual behavior
```
root@e64f53182357:/code/packages# ./build_packages.py --no-tetgen "${CODE_DIR}/packages"
Traceback (most recent call last):
  File "./build_packages.py", line 33, in <module>
    from freesurfer.log import *
  File "/code/python/freesurfer/__init__.py", line 3, in <module>
    from ._surface import *
  File "/code/python/freesurfer/_surface.py", line 1, in <module>
    import numpy as np
ImportError: No module named numpy
```
(after `pip install numpy`)
```
root@e64f53182357:/code/packages# ./build_packages.py --no-tetgen "${CODE_DIR}/packages"
Traceback (most recent call last):
  File "./build_packages.py", line 33, in <module>
    from freesurfer.log import *
  File "/code/python/freesurfer/__init__.py", line 5, in <module>
    from .freeview import fv
  File "/code/python/freesurfer/freeview.py", line 4, in <module>
    import nibabel as nib
ImportError: No module named nibabel
```
### Expected behavior
Ideally, no 3rd party modules should be required for the build script

### How to replicate the behavior
Use a python environment without `numpy` or `nibabel` installed, then run the `pacakges/build_packages.py` script, as described under "Making the third party packages" on the [wiki](https://surfer.nmr.mgh.harvard.edu/fswiki/CMake).

### Details
I'm trying to build the 3rd party libraries needed to build FreeSurfer, and it seems that `numpy` and `nibabel` are imported from within the `build_packages.py` script.  I'd like to not have these modules installed in my environment before installing FS itself. Alternatively, this could be mentioned within the wiki page.

**Comment by buildqa** [comment]
Hello.

I think this has since been remedied, i.e., I just ran the script in my built freesurfer tree on a Mac, and did not see any import errors.

**Comment by ltirrell** [comment]
Oh yes, forgot to close this. It was fixed in 66d7075. I have a PR (#559) that fixes it differently and includes other changes that may or may not be worth adding, but this issue is resolved.
